pkg/image/api/validation/validation.go

```diff
@@ -185,3 +183,45 @@ func ValidateImageStreamTagUpdate(newIST, oldIST *api.ImageStreamTag) fielderror
 
 	return result
 }
+
+func ValidateImageStreamImport(isi *api.ImageStreamImport) fielderrors.ValidationErrorList {
```

Comment:
We don't ignore one of them.  We return either.  However, we should return a validation error if you don't specify one or the other.